packages/functionals/botpress-qna/src/index.js

```diff
@@ -99,7 +99,15 @@ module.exports = {
        */
       shouldProcessMessage(fn) {
         shouldProcessMessage = fn
-      }
+      },
+
+      /**
+       * @async
+       * Returns question by id
+       * @param {String} id - id of the question to look for
+       * @returns {Object}
+       */
+      getQuestion: storage.getQuestion.bind(storage)
```

Comment:
Can you surface the `id` parameter? Interface leaking is bad

Reply:
What do you mean by surfacing parameter?